guides/common/modules/con_renewing-custom-ssl-certificate.adoc

```diff
@@ -0,0 +1,4 @@
+[id="renewing-custom-ssl-certificate_{context}"]
```

Comment:
Anchor, filename, and title do not match.

Reply:
Addressed :+1: